docs/user/lti.rst

```diff
@@ -0,0 +1,232 @@
+#. Navigate to Course Details: Once you're inside the course settings, scroll down to the Course Details section.
+#. Access LTI Configuration: Within the Course Details section, you'll find an link labeled LTI Configuration. Click on this to access the LTI settings specific to the course.
+#. Switch to the Exercises Tab: Inside the LTI Configuration, locate and click on the Exercises tab. This tab lists all the exercises available for the course.
+#. Retrieve the LTI 1.3 Launch URL: For each exercise listed, there's an associated LTI 1.3 Launch URL. This URL is crucial for linking the exercise to Moodle. Find the exercise you wish to link and copy its LTI 1.3 Launch URL.
+
+.. figure:: lti/ltiexercises_list.png
+    :align: center
+    :width: 700
+    :alt: LTI Exercise List
+
+2.2. Link exercise to Moodle
+""""""""""""""""""""""""""""
+
+With the LTI 1.3 Launch URL copied, you can now link the exercise in Moodle:
+
+#. Access the Desired Course: Log into your Moodle account and navigate to the course where you want to link the Artemis exercise.
+#. Enable Edit Mode: Once inside the course, turn on the Edit Mode. This mode allows you to make changes and add resources to the course.
+
+    .. figure:: lti/moodle_editmode.png
+        :align: center
+        :width: 700
+        :alt: Moodle - Edit Course
+
+#. Add an External Tool: Click on the Add an activity or resource button, which brings up a list of available activities and resources. From this list, select the External tool option.
+
+    .. figure:: lti/moodle_add_external_tool.png
+        :align: center
+        :width: 700
+        :alt: Moodle - Add External Tool
+
+#. Enter the LTI 1.3 Launch URL: In the settings for the external tool, you'll find a field labeled Resource URL. Paste the previously copied LTI 1.3 Launch URL from Artemis into this field.
+#. Save and Finalize: After entering the URL, click on the Save and return to course button to finalize the addition. The linked Artemis exercise should now be accessible directly from the Moodle course.
+
+.. figure:: lti/moodle_add_external_tool_page.png
+    :align: center
+    :width: 700
+    :alt: Moodle - Add External Tool Page
+
+
+LTI 1.3 Student Guide
+---------------------------
+For students, the integration of Artemis with Moodle via LTI 1.3 offers a streamlined experience to access and participate in Artemis exercises directly from the Moodle platform. The key steps involved are:
+
+#. Starting an Artemis Exercise: Students can easily access their Moodle courses and find the linked Artemis exercises. Clicking on these links will open the Artemis exercise page right within Moodle, providing a smooth transition.
+#. First-Time Users: If you're accessing an Artemis exercise for the first time, you'll receive a unique password. It's essential to note this password down as it will be required for future Artemis sessions.
```

Comment:
```suggestion
#. First-Time Users: If you are accessing an Artemis exercise for the first time, you will receive a unique password. It is essential to note this password down as it will be required for future Artemis sessions.
```